extra-resources/mods/Mythos/chinese.cardset

```diff
@@ -1,26 +1,14 @@
 package mythos
-/* See modding documentation for all available keywords. */
-/* GENERIC TEMPLATE
-card('ChangeMe') {
-    creature 'Chinese'
-    flavor 'hello'
-    health 1
-    sickness 1
-    manaCost 1
-    attack 1
-}
-*/
 
-/**
- * List of available cards for Chinese panthon for Cardshifter 'Mythos' mod
- * @author www.github.com/Phrancis
+/*
+ * List of available cards for Chinese pantheon for Cardshifter 'Mythos' mod.
  */
 
 // PANTHEON
 
-// Info: http://en.wikipedia.org/wiki/Jade_Emperor
-// Image: http://upload.wikimedia.org/wikipedia/commons/2/24/Jade_Emperor._Ming_Dynasty.jpg
-// License: Public Domain
+// https://en.wikipedia.org/wiki/Jade_Emperor
+// https://commons.wikimedia.org/wiki/File:Jade_Emperor._Ming_Dynasty.jpg
+// Attribution: By Anonymous [Public domain], <a href="https://commons.wikimedia.org/wiki/File%3AJade_Emperor._Ming_Dynasty.jpg">via Wikimedia Commons</a>
```

Comment:
The link to the image in the `href` is normal, that's generated by Wikimedia when the HTML option is selected. I figured if we ever wanted to be able to link the attribution directly into the client(s) we could just grab the anchor links from the file.